src/index.js

```diff
@@ -54,7 +54,7 @@ class Service {
     fromBuffer(buffer)
     .pipe(this.Model.createWriteStream({
       key: id,
-      params: params.query
+      params: params.s3
```

Comment:
Heh, the commit message should read 
>"Don't use the `query` object in favor of a Before hook"
Guess I can't use back ticks in my commit messages.